docs/content/onedrive.md

```diff
@@ -142,10 +165,6 @@ is 10MB.
 Note that OneDrive is case insensitive so you can't have a
 file called "Hello.doc" and one called "hello.doc".
 
-Rclone only supports your default OneDrive, and doesn't work with One
```

Comment:
Does this allow access to the non default OneDrive?  If not then then this needs amending slightly.